components/build/build-service/.tekton/pipelines-as-code.yaml

```diff
@@ -0,0 +1,7 @@
+apiVersion: pipelinesascode.tekton.dev/v1alpha1
+kind: Repository
+metadata:
+  name: build-service-pac
+  namespace: build-service-system
```

Comment:
We missed it last time. Should be consistent.

Reply:
Done